### ISSUE TITLE
[Stellar core database size] (Version: [stellar-core version])

### ISSUE THREAD

**Issue by HussienAbdelaal** [stellar/stellar-core#3355]
### Explain in detail the incorrect or additional documentation you'd like to see.

I need to know the storage requirements needed to fully sync a stellar core node with the blockchain. I already read the [documentation about compute requirements](https://developers.stellar.org/docs/run-core-node/prerequisites/#compute-requirements)  but its outdated. I need to know the current size of the stellar core database after being fully synced.
I already tried stellar exchange and [someone already asked a similar question a month ago with no answers](https://stellar.stackexchange.com/questions/5949/actual-blockchain-size).

I found an [outdated answer](https://stellar.stackexchange.com/a/153/5295) for what I'm looking for which was super helpful, but I need to know the current storage requirements needed to fully sync with the blockchain.

I also need to know the size of the horizon database to fully sync horizon with the blockchain and the size of the history archives for publishing the complete history.

Any help will be be much appreciated.
Thanks in advance.

### Please include any links to where our docs are hosted on our Developer Portal
https://developers.stellar.org/docs/run-core-node/prerequisites/#compute-requirements
https://developers.stellar.org/docs/run-api-server/prerequisites/#historical-catch-up

### Additional context
Basically if I can get an answer in the following format it would be great:
- Current total ledgers count: XXX
- Core database size (fully synced): XXX TB
- Horizon database size (fully synced): XXX TB
- History archives: XXX TB
